purchases/src/main/kotlin/com/revenuecat/purchases/common/events/BackendEvent.kt

```diff
@@ -0,0 +1,39 @@
+package com.revenuecat.purchases.common.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.customercenter.events.CustomerCenterBackendEvent
+import com.revenuecat.purchases.paywalls.events.PaywallBackendEvent
+import com.revenuecat.purchases.paywalls.events.PaywallEvent
+import com.revenuecat.purchases.utils.Event
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal sealed class BackendEvent : Event {
+    @Serializable
+    @SerialName("customer_center")
+    data class CustomerCenter(val event: CustomerCenterBackendEvent) : BackendEvent()
+
+    @Serializable
+    @SerialName("paywalls")
+    data class Paywalls(val event: PaywallBackendEvent) : BackendEvent()
```

Comment:
I find a bit confusing that `BackendEvent.Paywalls` contains a `PaywallBackendEvent`. Maybe it could be `PaywallBackendEventData`?